src/debuginfo.cpp

```diff
@@ -779,6 +798,7 @@ void jl_getFunctionInfo(const char **name, size_t *line, const char **filename,
 #endif
 #endif
         lookup_pointer(context, name, line, filename, pointer, 1, fromC);
+        delete context;
```

Comment:
I don't think so. The current scope starts from [here](https://github.com/JuliaLang/julia/pull/12433/files#diff-d4b80bf89e46982070f234dd7cc344a9R780) and the only possible values are `NULL`, `new DWARFContextInMemory(...)` or `DIContext::getDWARFContext`.